ethereum/contracts/BeefyLightClient.sol

```diff
@@ -285,13 +295,15 @@ contract BeefyLightClient {
         SimplifiedMMRProof calldata proof
     ) public {
         verifyCommitment(id, commitment, validatorProof);
+
+        bytes32 mmrRootHash = getMmrRootHash(commitment.payload);
         verifyNewestMMRLeaf(
             latestMMRLeaf,
-            commitment.payload,
```

Comment:
can we just keep using commitment.payload as is, based on my other suggestion below?